polaris-migrator/src/migrations/styles-replace-inline-comments/transform.ts

```diff
@@ -0,0 +1,37 @@
+import type {API, FileInfo, Options} from 'jscodeshift';
+import type {Plugin} from 'postcss';
+import postcss from 'postcss';
+import stylelint from 'stylelint';
+
+const plugin = (): Plugin => {
+  return {
+    postcssPlugin: 'styles-replace-inline-comments',
+    Comment(comment) {
+      comment.raws.inline = false;
+      comment.raws.right = ' ';
+      comment.raws.left = ' ';
+    },
```

Comment:
```suggestion
    Comment(comment) {
      comment.raws.before = '\n';
      comment.raws.inline = false;
      comment.raws.right = ' ';
      comment.raws.left = ' ';
    },
```

GPT suggested using `comment.raws.before` 🤔 . Maybe conditionally depending on the current value of `comment.raws.before`, let's say if it's already `\n`.

Reply:
Okay, so looked at this briefly yesterday and I suspect it's a bit more involved than this, because sometimes we want new break lines and some times we don't, and it depends on the `prev()` node.

I need to take a closer look to fine tune this, so I'll circle back to this in a few days 🤞